website/docs/cloud-docs/api-docs/configuration-versions.mdx

```diff
@@ -63,6 +63,8 @@ A configuration version created through the API or CLI can only be used in runs
 | `uploaded`             | The configuration version is fully processed and can be used in runs.                                                                                                                                                                                                                                    |
 | `archived`             | All immediate runs are complete and Terraform Cloud has discarded the files associated with this configuration version. If the configuration version was created through a connected VCS repository, it can still be used in new runs. In those cases, Terraform Cloud will re-fetch the files from VCS. |
 | `errored`              | Terraform Cloud could not process this configuration version, and it cannot be used to create new runs. You can try again by pushing a new commit to your linked VCS repository or creating a new configuration version with the API or CLI.                                                             |
+| `backing_data_soft_deleted`  | Terraform Enterprise has soft-deleted the backing files associated with this configuration version. You can restore the backing data associated with it before it is permanently deleted if need be. |
```

Comment:
A rare opportunity for passive voice: technically, we don't know that it was TFE (via data retention policy that soft deleted things), since users can soft delete things too. So, you might want to say something like "The configuration version's backing data has been soft deleted..."

Reply:
Great call out! I've updated this ✔️